java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Updated jakarta-activation reference (jaf).
```

Comment:
As this is an internal-only change I believe this should not really be documented in the changelog

Reply:
Removed it. So change log is only for functional change, not for technical changes?

Reply:
We try to make these changelogs usefull for the end users/administrators. So we try to write them in a way that non developers can understand them.

Developers can read the git history.